docs/sources/flow/getting-started/migrating-from-static.md

```diff
@@ -297,15 +297,41 @@ loki.write "logs_varlogs" {
 }
 ```
 
+## Integrations Next
+
+[Integrations next][] configs can be converter or run by adding the `extra-args` flag for [convert][] or `config.extra-args` for [run][].
```

Comment:
```suggestion
You can convert the [integrations next][] configurations, or you can add the `extra-args` flag for [convert][] or `config.extra-args` for [run][] to use the configurations as-is.
```

Reply:
```suggestion
You can convert [integrations next][] configurations by adding the `extra-args` flag for [convert][] or `config.extra-args` for [run][].
```

Reply:
I applied something a little different since I think my intended meaning got jumbled with your improved phrasing.

Reply:
Looks MUCH better now with the "something a little different" and now it makes sense :-)